src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
+    {
+        public ILogger Logger { get; set; }
```

Comment:
Why is ILogger not a required ctor argument?  Won't everything blow up if its null, there isn't any null checks in the methods currently